site/middleware.ts

```diff
@@ -0,0 +1,35 @@
+import { geolocation, next } from '@vercel/edge';
+
+// List of blocked countries (ISO 3166-1 alpha-2 country codes)
+// North Korea (KP), Iran (IR), Syria (SY), Cuba (CU), Crimea (UA-43), Luhansk (UA-09), Donetsk (UA-14)
+const BLOCKED_COUNTRIES = [
+  'KP',
+  'IR',
+  'SY',
+  'CU',
+  'UA-43',
+  'UA-09',
+  'UA-14',
+  'UNKNOWN',
+];
+
+export const config = {
+  matcher: ['/'],
+};
```

Comment:
Currently set to run on the entire site

Reply:
If we'd like to test a single endpoint in prod before rolling out to the entire site then I can update this matcher to be `/swap/swap`

Reply:
Perfect, can you set the matcher to a demo-page. So we can merge this.

Reply:
Done! Set to a demo page at /temp-demo